docs/efun/interactive/has_gmcp.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: interactive / has_gmcp
+---
+
+### NAME
+
+    has_gmcp() - detects whether or not a given interactive user's client 
```

Comment:
I would not say it “detects” , just “returns”

this function doesn’t trigger gmcp negotiation, it just tells if this client has gmcp enabled

Reply:
fixed in 2690f8b